cloudinstall/utils.py

```diff
@@ -771,6 +771,25 @@ def download_url(url, output_file):
             url, res.content))
 
 
+def pollinate(session, tag):
+    """ fetches random seed
+
+    :param str session: randomly generated session id
+    :param str tag: custom tag
+    """
+    if not os.path.isfile('/usr/bin/pollinate'):
+        return
+    if tag not in ['IL', 'IS', 'IM', 'DL', 'DM', 'DS']:
+        raise UtilsException("Unknown TAG {}".format(tag))
+    agent_str = 'uoi/{}-{}'.format(session, tag)
```

Comment:
since uuid4 creates strings with lots of dashes, might be cleaner to make agent_str be `'uoi/{}/{}'` instead - it'd be easier to parse on the other end